ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -777,4 +777,7 @@ literals:
 # Added for TW
 - supply_air_isolation_damper_closed_status
 - supply_air_isolation_damper_open_status
-- exhaust_fan_speed_frequency_command
\ No newline at end of file
+- exhaust_fan_speed_frequency_command
+- pump_speed_frequency_command
```

Comment:
Not valid. It is clear from the type whether the device is a pump or not. The only time pump would be valid is if it is describing an integral component of a device (e.g. a circulation_pump_run_status on a boiler or a spray_pump_run_command on a cooling tower). Please remove.

Reply:
It was removed